src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_RepoPolicies.cs

```diff
@@ -0,0 +1,44 @@
+using System.Threading.Tasks;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    public class ScenarioTests_RepoPolicies : MaestroScenarioTestBase
+    {
+        private readonly string repoName = "maestro-test1";
+        private readonly string branchName = "RepoPoliciesTestBranch";
+
+        [Test]
+        public async Task ArcadeRepoPolicies_EndToEnd()
+        {
+            TestContext.WriteLine("Repository merge policy handling");
+            TestContext.WriteLine("Running tests...");
+
+            string repoUrl = GetRepoUrl(repoName);
+
+            TestContext.WriteLine("Setting repository merge policy to empty");
```

Comment:
If we put this string into the assert on line 25, it will make the assert more helpful when it triggers. :-)